ivy/functional/frontends/jax/devicearray.py

```diff
@@ -1,18 +1,79 @@
+# global
+import functools
+
+
 # local
 import ivy
 import ivy.functional.frontends.jax as jax_frontend
 
 
+def _frontend_wrapper(f):
+    @functools.wraps(f)
+    def decor(self, *args, **kwargs):
+        args = list(args)
+        if isinstance(self, jax_frontend.DeviceArray):
+            i = 0
+            for arg in args:
+                if isinstance(arg, jax_frontend.DeviceArray):
+                    args[i] = arg.data
+                i += 1
+            for u, v in kwargs:
```

Comment:
iterating through a dict only gives its keys, you would need `kwargs.items()`

Reply:
ah right, the very last commit for this fix was in my local, didn't know it's not pushed yet 😅 just pushed it, sorry!